zjit/src/backend/lir.rs

```diff
@@ -2243,6 +2243,15 @@ macro_rules! asm_comment {
 }
 pub(crate) use asm_comment;
 
+/// Convenience macro over [`Assembler::ccall`] that also adds a comment with the function name.
+macro_rules! ccall {
+    [$fn_name:ident ( $($args:expr),* ) , $asm: ident] => {{
```

Comment:
Can this instead use the `asm` from the existing scope?

Reply:
I'm usually the guy who suggests that kind of implicit stuff to make things shorter, but we will not be able to use such a macro in `compile_side_exits()` where `asm` is called `self` (unless you add `let asm = self;` for the C call).